update-wasm-grammars.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd web-playground/public/scripts/
+
+WORKDIR=$(mktemp -d)
+BLUE='\033[0;34m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+echo -e "${BLUE}Updating all Topiary grammars. This process can take a few minutes."
+
+json() {
+  echo -e "${BLUE}JSON: Fetching${NC}"
+  git clone --depth=1 https://github.com/tree-sitter/tree-sitter-json.git "${WORKDIR}/tree-sitter-json" &> /dev/null
```

Comment:
Maybe the repositories should be fetched from a config file?
Same remark for all other functions.

Reply:
Making this general and driving it from config definitely seems worth doing. Using `Cargo.toml` as the config source would be pretty cute and ensure the WASM blobs match the known-to-work revisions. (Of course, some additional source of config would be needed to handle the exceptions; i.e., OCaml.)

Reply:
It now uses the `Cargo.lock` file to select the correct grammars.

Reply:
With the additional handling needed for grammars like OCaml, I'm leaving it at this for now.